### PR TITLE
Rewrite about-us page. Add more bio for Thomas, but does not work cor…

### PR DESCRIPTION
**Pull request by twiecki**
…rectly yet.

### CODE REVIEW

File: content/team/thomas-wiecki/contents.lr [hunk 1]
```diff
@@ -10,7 +10,14 @@ linkedin_url: https://www.linkedin.com/in/thomas-wiecki-46339244/
 ---
 personal_url: https://twiecki.io
 ---
-blurb: Making Bayesian modeling accessible to a wider audience.
+blurb: 
+* PyMC3 author
+* PhD on computational cognitive neuroscience from Brown University 
+* Former VP of data science and head of research at [Quantopian Inc](https://quantopian.com): building a team of data scientists to build a hedge fund from a pool of 300k crowd researchers.
+* Recognized public speaker: keynotes at the Open Data Science Conference (ODSC) & TACC as well as talks at Strata Hadoop & AI, Newsweek AI conference, and various PyData conferences around the world
+* Runs a [data science podcast](http://www.pydata-podcast.com/), [blogs about Bayesian statistics](https://twiecki.io), and is an [avid Twitter personality](https://twiecki.io).
```
Comment (OriolAbril) [inline content/team/thomas-wiecki/contents.lr]: the twitter link is also to the website, not to twitter profile

**Review by OriolAbril — CHANGES_REQUESTED** [review]
It looks like you forgot to add the `content/what-we-do/contents.lr` file to git. Right now instead of moved from about us to what we do the file is deleted.

**Comment by twiecki** [comment]
Thanks for reviewing, fixed.

File: content/what-we-do/contents.lr [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+----
+section_level: 3
+----
+markdown: 
+
+Another benefit of Bayesian statistics is that it allows you to incorporate the structure of your data into the model directly. This is different from machine learning which has to infer all structure from data -- there is no way to inform these models of structure ahead of time, which is why they require so much data.
+
+When working with us we first gain a deep understanding of your data structure and the specific problem you want to solve. Together with you, we tailor a custom model that solves your specific problem and takes your unique data structure into account.
+
+The resulting model requires far fewer data points than in machine learning. For example, many data sets have a nested or hierarchical structure that is impossible to map adequately in a machine learning algorithm, but is very [naturally supported by PyMC3]((https://twiecki.io/blog/2014/03/17/bayesian-glms-3/)).
```
Comment (OriolAbril) [inline content/what-we-do/contents.lr]:
```suggestion
The resulting model requires far fewer data points than in machine learning. For example, many data sets have a nested or hierarchical structure that is impossible to map adequately in a machine learning algorithm, but is [naturally supported by PyMC3]((https://twiecki.io/blog/2014/03/17/bayesian-glms-3/)).
```

File: content/what-we-do/contents.lr [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    * Other non-trivially structured data sets
+* Your problem is not a simple prediction problem
+    * If you have a large set of tabular data with labels and only care about prediction accuracy, machine learning will be the best tool
+    * If you care about more than prediction, Bayesian modeling will be a good fit
+* You make decisions with real-world consequences based on noisy data
+    * In this case it is important to take all possible interpretations of your data into account, not just the most likely
+    * Bayesian statistics naturally supports this by modeling beliefs as distributions rather than scalar values
+    * This results in more robust decisions that work across a whole range of plausible assumptions
+
+Problems in many domains meet these criteria as demonstrated by our customers in diverse fields ranging from finance, biotech, agriculture, pharma, adtech and more.
```
Comment (OriolAbril) [inline content/what-we-do/contents.lr:128]: can we add a link to testimonials here?

File: content/what-we-do/contents.lr [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+If you are interested in learning more, please email us at [thomas.wiecki@pymc-labs.io](mailto:thomas.wiecki@pymc-labs.io). We look forward to hearing from you.
```
Comment (OriolAbril) [inline content/what-we-do/contents.lr:130]: We should also update https://github.com/pymc-labs/pymc-labs.github.io/blob/master/databags/contact.json to show the pymc-labs mail on the home page.

**Review by OriolAbril — APPROVED** [review]
This is great!

File: content/what-we-do/contents.lr (hunk 4 above)
Comment (twiecki) [reply content/what-we-do/contents.lr:130]: Done.

**Pull request merged**